lib/kitchen/provisioner/chef_base.rb

```diff
@@ -127,6 +127,10 @@ class ChefBase < Base
 
       default_config :architecture
 
+      default_config :download_url
+
+      default_config :checksum
```

Comment:
Is any checksum type supported?

Reply:
sha256 only since that's what the install scripts validate. This will be documented.